core/src/main/java/brooklyn/event/feed/windows/WindowsPerformanceCounterFeed.java

```diff
@@ -251,10 +258,71 @@ public T call() throws Exception {
         }
     }
 
+    private static class SendPerfCountersToSensors implements PollHandler<WinRmToolResponse> {
+
+        private final EntityLocal entity;
+        private final List<WindowsPerformanceCounterPollConfig<?>> polls;
+
+        public SendPerfCountersToSensors(EntityLocal entity, List<WindowsPerformanceCounterPollConfig<?>> polls) {
+            this.entity = entity;
+            this.polls = polls;
+        }
+
+        @Override
+        public boolean checkSuccess(WinRmToolResponse val) {
+            if (val.getStatusCode() != 0) return false;
+            String stderr = val.getStdErr();
+            if (stderr == null || stderr.length() != 0) return false;
```

Comment:
Status code is unreliable (it seems, empirically) - returns 0 sometimes even when failed (but never returns non-zero on success; but if the connection drops, then we'll get an exception though the command might continue executing on the Windows machine to completion).